test/aglint/list3.txt

```diff
@@ -0,0 +1,20 @@
+! By default, AGLint should show a warning for multiple selectors in a single rule
+example.com##.ad1, .ad2

```

Comment:
Does not show any warning with the default configuration.

Reply:
Nit, I removed `single-selector` rule from recommended rules. It is only included in `aglint:all` preset

Reply:
Fixed in https://github.com/ameshkov/VscodeAdblockSyntax/pull/63/commits/bd0187355d1840eed20f697cb434b9a19fda933d